primitive-types/src/lib.rs

```diff
@@ -22,6 +22,7 @@ use fixed_hash::{construct_fixed_hash, impl_fixed_hash_conversions};
 #[cfg(feature = "scale-info")]
 use scale_info::TypeInfo;
 use uint::{construct_uint, uint_full_mul_reg};
+pub use uint::{FromStrRadixErr, FromStrRadixErrKind};
```

Comment:
hmm, I overlooked this line, seems redundant
we could return `uint::FromStrRadixErr` in `impls/num-traits` instead of `$crate::...`

Reply:
I'm not super familiar with the macro hygiene rules, but what if the downstream crate did a `extern crate uint as floop`?

Reply:
yeah, fair point, we can do what `impl/rlp` does, but still this like seems redundant